csharp/ql/test/library-tests/csharp9/PrintAst.expected

```diff
@@ -0,0 +1,284 @@
+Record.cs:
+#    4| [NamespaceDeclaration] namespace ... { ... }
+#    6|   1: [Class] IsExternalInit
+#    9| [Class] Person
+#    9|   11: [Property] EqualityContract
+#    9|     3: [Getter] get_EqualityContract
+#    9|   11: [NEOperator] !=
+#-----|     2: (Parameters)
+#    9|       0: [Parameter] r1
+#    9|       1: [Parameter] r2
+#    9|   11: [EQOperator] ==
+#-----|     2: (Parameters)
+#    9|       0: [Parameter] r1
+#    9|       1: [Parameter] r2
+#   11|   14: [IndexerProperty] LastName
```

Comment:
It was probably a mistake to show `IndexerProperty` at all in the tree. I didn't realize this before. Should we show simply `Property` instead?

Reply:
Created a separate PR for this: https://github.com/github/codeql/pull/5063